docs/source/AdministratorGuide/Systems/MonitoringSystem/index.rst

```diff
@@ -23,8 +23,7 @@ Install Elasticsearch
 ======================
 
 This is not covered here, as installation and administration of ES are not part of DIRAC guide.
-Just a note on the ES versions supported: ES7 and ES6 are supported, the support for ES5 is not assured,
-and the one for ES6 will be dropped in a future release.
+Just a note on the ES versions supported: only ES7+ versions are currently supported, and are later to be replaced by OpenSearch services.
```

Comment:
> And update the documentation (somewhere it's written which ES versions are supported).

On this part of the documentation for the integration version of DIRAC, do we want to still support ES7+ or do we just keep OpenSearch? Meaning, shall I remove the mention of ES here, since we will be using only OS services when this version will be released?

Reply:
Let's decide the plan at the next BiLD meeting in 2 days.

Reply:
Okay, I will keep this PR as draft until then.